### ISSUE TITLE
Add Fuchsia accessibility protocol to accessibility API section.

### ISSUE THREAD

**Issue by jasonjgw** [w3c/a11y-discov-vocab#24]
Consider adding the Fuchsia accessibility protocol to the accessibility APIs section (currently section 2.2).
https://fuchsia.dev/reference/fidl/fuchsia.accessibility

**Comment by mattgarrish** [comment]
Is there a formal name for their API?

"fuschia.accessibility" sounds very SDK-ish.

**Comment by jasonjgw** [comment]
Unfortunately, I don't know anything other than what appears on their Web page as cited above.

**Comment by clapierre** [comment]
@jasonjgw looks like they have already modified the github repo and that folder you pointed to no longer exists.
Looks like there is a .semantics / .gestures and .keyboard sub directories now but the root folder of just .accessibility doesn't exist.

**Comment by mattgarrish** [comment]
There's an overview of the framework here: https://fuchsia.dev/fuchsia-src/concepts/accessibility/accessibility_framework

I suppose there's no formal "BlackBerryAccessibility" API, so maybe the precedent would be to call it FuschiaAccessibility.

**Comment by mattgarrish** [comment]
There seem to be at least a few implementations of Fuchsia now, but documentation is a bit thin. The framework page I cited above says:

> Fuchsia's accessibility framework is primarily implemented in the Accessibility Manager.

Makes me think we should call this `AccessibilityManager` if we implement it.
